CHANGES/10759.packaging.rst

```diff
@@ -0,0 +1,5 @@
+Support building against system llhttp library -- by :user:`mgorny`
+
+This change adds support for ``USE_SYSTEM_DEPS`` environment variable that
```

Comment:
If this is documented in `index.rst` under `.. envvar::`, we could reference it via 
```suggestion
This change adds support for :envvar:`USE_SYSTEM_DEPS` environment variable that
```

Reply:
Ah, sorry, I misunderstood your comment. However, I've documented both `AIOHTTP_*` variables in glossary now (though I'm not 100% sure that's the best place to put them).